pyroaring/bitmap.pxi

```diff
@@ -116,6 +116,37 @@ cdef class BitMap(AbstractBitMap):
             else:
                 self &= AbstractBitMap(values, copy_on_write=self.copy_on_write)
 
+    def difference_update(self, *others):
+        """
+        Remove all elements of another set from this set.
+        """
+        self.__isub__(AbstractBitMap.union(*others))
+
+    def symmetric_difference_update(self, other):
+        """
+        Update a set with the symmetric difference of itself and another.
+        """
+        union = self & other
+        self.__ior__(other)
+        self.__isub__(union)
+
+    def clear(self):
+        """Remove all elements from this set."""
+        self.__isub__(self)
```

Comment:
You should use [roaring_bitmap_clear](https://github.com/RoaringBitmap/CRoaring/blob/995cabc6ff179cefd930cf68c35d5ddd8df112eb/include/roaring/roaring.h#L330) function.